app/policies/placements/mentor_policy.rb

```diff
@@ -3,7 +3,13 @@ class Scope < ApplicationPolicy::Scope
     def resolve
       return scope if user.support_user?
 
-      scope.joins(:mentor_memberships).where(mentor_memberships: { school: user.current_organisation })
+      scope.where(
+        mentor_memberships: { school: user.current_organisation },
+      ).or(
+        scope.where(
+          id: Placements::MentorMembership.select(:mentor_id).where(school: user.current_organisation),
+        ),
+      )
```

Comment:
Why do we have an `OR` condition in this query now? 🤔 Shouldn't it work with just the subquery?

Right now it looks like we're doubling up with the same `WHERE` conditions twice.

```suggestion
      scope.where(
        id: Placements::MentorMembership.select(:mentor_id).where(school: user.current_organisation),
      )
```